modules/perception/lidar/lib/detection/lidar_point_pillars/include/point_pillars_detection.h

```diff
@@ -55,15 +57,21 @@ class PointPillarsDetection {
   // reference pointer of lidar frame
   LidarFrame* lidar_frame_ref_ = nullptr;
   std::shared_ptr<base::AttributePointCloud<base::PointF>> original_cloud_;
-  int gpu_id_ = -1;
+  std::shared_ptr<base::AttributePointCloud<base::PointD>>
+      original_world_cloud_;
+  int gpu_id_ = 0;
 
   // PointPillars
   std::unique_ptr<PointPillars> point_pillars_ptr_;
   bool reproduce_result_mode_ = false;
   float score_threshold_ = 0.5;
   float nms_overlap_threshold_ = 0.5;
-  std::string pfe_onnx_file_;
-  std::string rpn_onnx_file_;
+  std::string pfe_onnx_file_ =
+      "/apollo/modules/perception/lidar/lib/detection/lidar_point_pillars/"
+      "test/data/pfe.onnx";
+  std::string rpn_onnx_file_ =
+      "/apollo/modules/perception/lidar/lib/detection/lidar_point_pillars/"
+      "test/data/rpn.onnx";
 
```

Comment:
If the two files are not for testing, move them out of test folder.

Reply:
Have copied them into perception/production/data/models, but they need updated to a better trained version soon. And original files are still requested by the unit test.